app/main.py

```diff
@@ -1,3 +1,57 @@
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int:
+        if type(other) == int or type(other) == float:
```

Comment:
In annotation you use only int, but here and below in other methods you check float